modules/phase_field/src/userobjects/DiscreteNucleationInserter.C

```diff
@@ -79,6 +79,7 @@ DiscreteNucleationInserter::threadJoin(const UserObject &y)
   // combine _local_nucleus_list entries from all threads on the current process
   const DiscreteNucleationInserter & uo = static_cast<const DiscreteNucleationInserter &>(y);
   _local_nucleus_list.insert(_local_nucleus_list.end(), uo._local_nucleus_list.begin(), uo._local_nucleus_list.end());
+  _changes_made += uo._changes_made;
```

Comment:
I've seen this pattern in framework. I wasn't sure if that works (i.e. If all threads sum this up and hand out the already summed up value won't this overcount and have a race condition?!)